!GameTexts/NewTexts.xml

```diff
@@ -125,32 +121,30 @@
 			<Text>Mingghan</Text>
 		</Row>
 		<Row Tag="TXT_KEY_PROMOTION_UNIT_MONGOLIA_MINGGHAN_HELP">
-			<Text>+5 XP [COLOR_POSITIVE_TEXT]on Pillage[ENDCOLOR]. +20% [ICON_STRENGTH] Combat Strength if unit is standing [COLOR_POSITIVE_TEXT]on Pillaged Tile[ENDCOLOR].</Text>
+			<Text>+5 XP [COLOR_POSITIVE_TEXT]on Pillage[ENDCOLOR].[NEWLINE]+20% [ICON_STRENGTH] Combat Strength if unit is standing [COLOR_POSITIVE_TEXT]on Pillaged Tile[ENDCOLOR].</Text>
 		</Row>
 		<Row Tag="TXT_KEY_PROMOTION_UNIT_MONGOLIA_MASSACRE">
 			<Text>Massacre</Text>
 		</Row>
 		<Row Tag="TXT_KEY_PROMOTION_UNIT_MONGOLIA_MASSACRE_HELP">
-			<Text>+20% [ICON_STRENGTH] Combat Strength [COLOR_POSITIVE_TEXT]from Pillaged Tile[ENDCOLOR].</Text>
+			<Text>+20% [ICON_STRENGTH] Combat Strength if unit is standing [COLOR_POSITIVE_TEXT]on Pillaged Tile[ENDCOLOR].</Text>
 		</Row>
 		<Row Tag="TXT_KEY_UNIT_MONGOLIA_BLACK_TUG">
 			<Text>Black Tug</Text>
 		</Row>
 		<Row Tag="TXT_KEY_UNIT_MONGOLIA_BLACK_TUG_HELP">
-			<Text>Aggressive and menacing Medieval Era Mounted Unit. Only the Mongols may build it. Has no penalty against Cities and is stronger against wounded units or while standing on pillaged tiles. Gains additional XP for pillaging.</Text>
+			<Text>Aggressive and menacing Medieval Era Mounted Unit. Only the Mongols may build it. ties and is stronger against wounded units or while standing on pillaged tiles. Gains additional XP for pillaging.</Text>
```

Comment:
"ties" without capital.

Reply:
look like a sloppy delete on my part. change to 
"Only the Mongols may build it. Stronger against wounded units..."